code/game/objects/structures/ai_core.dm

```diff
@@ -24,11 +26,60 @@
 		brain = null
 	return ..()
 
+/obj/structure/AIcore/latejoin_inactive
+	name = "Networked AI core"
+	desc = "This AI core is connected by bluespace transmitters to NTNet, allowing for an AI personality to be downloaded to it on the fly mid-shift."
+	can_deconstruct = FALSE
+	icon_state = "ai-empty"
+	anchored = TRUE
+	state = AI_READY_CORE
+	var/available = TRUE
+	var/safety_checks = TRUE
+	var/active = TRUE
+
+/obj/structure/AIcore/latejoin_inactive/examine(mob/user)
+	. = ..()
+	to_chat(user, "Its transmitter seems to be [active? "on" : "off"].")
+
+/obj/structure/AIcore/latejoin_inactive/proc/is_available()			//If people still manage to use this feature to spawn-kill AI latejoins ahelp them.
+	if(!available)
+		return FALSE
+	if(!safety_checks)
+		return TRUE
+	if(!active)
+		return FALSE
+	var/turf/T = get_turf(src)
+	var/area/A = get_area(src)
+	if(!A.blob_allowed)

```

Comment:
Why this ? Core's don't care about this.

Reply:
It's a general way to see if the AI should be allowed to be put there to be downloaded into. If a blob core shouldn't be allowed to be there for balance reasons, the latejoin AI shouldn't be either, as it'll have immediate access to the entire station.